lib/trino-filesystem-azure/src/test/java/io/trino/filesystem/azure/TestAzureLocation.java

```diff
@@ -64,14 +64,20 @@ void test()
         assertInvalid("abfs://container@account.fake.dfs.core.windows.net/some/path/file");
     }
 
-    private static void assertValid(String uri, String expectedAccount, String expectedContainer, String expectedPath)
+    private static void assertValid(String uri, String expectedAccount, String expectedContainer, String expectedPath, Optional<String> scheme)
```

Comment:
We could rename to `expectedScheme` and change the type to non-optional `String` as the expected value isn't nullable.